cylc/flow/unicode_rules.py

```diff
@@ -105,6 +105,24 @@ def disallowed_characters(*chars):
     )
 
 
+def disallowed_string(string):
+    """Prevent use of an exact string.
+
+    Examples:
+        >>> regex, message = disallowed_string("I'm Batman.")
+        >>> message
+        "cannot be the string ``I'm Batman.``"
+        >>> bool(regex.match("I\'m Batman."))
+        True
+        >>> bool(regex.match("I'm Batmans"))
+        False
+    """
+    return (
+        re.compile(fr'^{re.escape(string)}$'),
+        f'cannot be the string ``{string}``'
+    )
+
+
```

Comment:
Did this sneak in from another branch?

Reply:
I thought I needed this to further constrain custom task output messages. Then, as soon as I finished writing this function I realised I had got the `satisfied` and `output` columns from the `task_prerequisites` table mixed up in my head, so this wasn't needed. Left it in in case it ends up being useful in future?

Reply:
This will get identified as dead code once we get code coverage back.  Better to hide it in a comment if it's worth keeping?

Reply:
We try to avoid [introns](https://en.wikipedia.org/wiki/Intron), generally better to have the bare minimum of code in the repo, easy to revive from a branch if needed in the future.